board.py

```diff
@@ -6,7 +11,41 @@ class Stone(Enum):
     EMPTY = 0
 
 class Board:
+    """
+    This is a class to hold our game board object.
+    It doesn't matter how it is implemented as long as 
+    calling board.put_stone_at(1,1, Stone.BLACK) will later mean that
+    board.get_stone_at(1,1) == Stone.BLACK
+    """
+    def __init__(self, width, height):
+        """
+        This is called an initialiser. If you need to do some setup when the 
+        board is created with board = Board(), do it here.
+        perhaps making something to store the stones in might be worthwhile?
+        """
+        self.boardStone = [[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY],[Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY,Stone.EMPTY]]
```

Comment:
You should replace this with something less verbose.

`[[Stone.EMPTY] * width for range(height)]`

Is one option.

Or being less tricky.
self.boardStone = []
```
for y in range(height):
    self.boardStone.push([])
    for x in range(width):
        self.boardStone[y].push(Stone.Empty)
```